website/src/components/animation-loop-runner.jsx

```diff
@@ -74,7 +79,7 @@ export default class AnimationLoopRunner extends Component {
 
   componentWillUnmount() {
     this.animationLoop.stop(this.props);
-    this.animationLoop.finalize();
+    this.animationLoop.delete();
```

Comment:
What's the issue with `finalize()`?

Reply:
> What's the issue with finalize()?

For historic reasons, deck.gl uses `finalize()`. luma.gl uses `delete()`.